modules/src/ics04_channel/msgs/chan_open_init.rs

```diff
@@ -23,6 +23,28 @@ pub struct MsgChannelOpenInit {
     pub signer: AccountId,
 }
 
+impl MsgChannelOpenInit {
+    /// Getter: borrow the `connection_id` from this message.
+    pub fn channel_id(&self) -> &ChannelId {
+        &self.channel_id
+    }
+
+    /// Getter: borrow the `client_id` from this message.
+    pub fn port_id(&self) -> &PortId {
+        &self.port_id
+    }
+
+    /// Getter: borrow the `counterparty` from this message.
+    pub fn channel(&self) -> &ChannelEnd {
+        &self.channel
+    }
+
+    // /// Setter for `connection_id`. Amenable to chaining, since it consumes the input message.
```

Comment:
Can you explain why this is commented out?

Reply:
uncertain if we need a DS for connection -> channel because of confusing spec ICS04 :). 
The last query in ICS04 takes connection as parameter and I wonder if the intend is to retrieve all channels open over a certain connection. Waited to discuss with you ... but now I see it as copy paste  mistake in ICS04 queryChannel should have been portId instead of connId. :)